test/mechanics/cell/PatchTest.cpp

```diff
@@ -188,6 +192,24 @@ BOOST_AUTO_TEST_CASE(PatchTestForce)
             node.SetValue(1, newDisplacements[dofY]);
     }
 
+    int pointsPerDirection = std::lround(std::sqrt(integrationTypeBc.GetNumIntegrationPoints()));
+    pointsPerDirection += 1; // one point per direction doesn't do much Voronoiying
+    Visualize::Visualizer<Visualize::TensorProductVoronoiHandler<2>> visualize(momentumBalanceCells,
+                                                                               pointsPerDirection);
+    visualize.DofValues(displ);
+
+    auto stress = [linearElasticLaw, displ](const CellData& cellData, const CellIpData& cellIpData) {
+        EngineeringStrain<2> strain = cellIpData.GetBMatrixStrain(displ) * cellData.GetNodeValues(displ);
+        return linearElasticLaw.Stress(strain, 0.0, 0, 0);
+    };
+    visualize.CellData(stress, "Stress");
+
+    visualize.CellData([](const CellData&, const CellIpData&) { return Eigen::Matrix<double, 1, 1>(7.0); }, "Seven");
+    visualize.CellData(
+            [](const CellData& cd, const CellIpData&) { return Eigen::Matrix<double, 1, 1>(cd.GetCellId()); },
+            "CellId");
+    visualize.WriteVTKFile("outputVoronoi.vtu");
```

Comment:
1) `WriteVtkFile` would match our naming convention. 
2) VTK or VTU? VTU is the XML version of VTK. The method name suggests that the output file should be of type `*.vtk`. Since we actually write VTU, paraview expects the wrong format. So I suggest renaming the method to `WriteVtuFile()`. And adding a check like `assert(filename.EndsWith(".vtu")`.